src/prisma/generator/models.py

```diff
@@ -347,6 +350,17 @@ def validate_version(cls, values: Dict[Any, Any]) -> Dict[Any, Any]:
         return values
 
 
+class BinaryPaths(BaseModel):
+    query_engine: Dict[str, str] = FieldInfo(
```

Comment:
Maybe a comment here about what this mapping is would be useful? I assume its something that platform to filename of the engine? An alternative to not using code would be a semantic type alias that resolves to str but is named something meaningful like `PLATFORM`. Or we can use enum as key?

Reply:
We would probably be able to use an enum as a key but we do not currently have the architecture that resolves path names properly. I also think there may be cases where the generated platform name isn't always limited to a certain set of enum values.

Added documentation to this in a785d7c49e6a05ed04994fe81d1c3d4fa1aacb88.